packages/firestore/src/local/memory_query_cache.ts

```diff
@@ -65,15 +67,29 @@ export class MemoryQueryCache implements QueryCache {
     return PersistencePromise.resolve();
   }
 
-  addQueryData(
-    transaction: PersistenceTransaction,
-    queryData: QueryData
-  ): PersistencePromise<void> {
+  private saveQueryData(queryData: QueryData) {
     this.queries.set(queryData.query, queryData);
     const targetId = queryData.targetId;
     if (targetId > this.highestTargetId) {
       this.highestTargetId = targetId;
     }
+    // TODO(gsoltis): track sequence number
```

Comment:
Consider doing TODO(GC) or similar instead so that we can grep for all of these once GC is "done" and verify nothing was missed. This is what we've done for other features in the past (and Sebastian is doing TODO(multitab) for multi-tab, etc).

Reply:
Sure.